### PR TITLE
Kokkos: update package for release 3.6.00

### PR DESCRIPTION
**Pull request by lucbv**
Updating the package slightly according to new release 3.6.00

### CODE REVIEW

File: var/spack/repos/builtin/packages/kokkos/package.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ class Kokkos(CMakePackage, CudaPackage, ROCmPackage):
 
     homepage = "https://github.com/kokkos/kokkos"
     git      = "https://github.com/kokkos/kokkos.git"
-    url      = "https://github.com/kokkos/kokkos/archive/3.5.00.tar.gz"
+    url      = "https://github.com/kokkos/kokkos/archive/3.6.00.tar.gz"
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/kokkos/package.py:18]: FYI. Changing the version here isn't necessary as Spack will automatically perform that conversion for you.

**Review by tldahlgren — APPROVED** [review]
Confirmed the sha256.  Deferring decision to merge to the requested reviewer.

**Comment by janciesko** [comment]
LGTM, thanks @lucbv.

**Review by crtrott — APPROVED** [review]

**Comment by lucbv** [comment]
@tldahlgren I think this is ready to merge : )

**Pull request merged**